src/bptl/conf/base.py

```diff
@@ -397,8 +399,30 @@
 # Celery
 CELERY_BROKER_URL = os.getenv("CELERY_BROKER_URL", "redis://localhost:6379/0")
 CELERY_RESULT_BACKEND = os.getenv("CELERY_RESULT_BACKEND", "redis://localhost:6379/0")
+
 # Add a 30 minutes timeout to all Celery tasks.
 CELERY_TASK_SOFT_TIME_LIMIT = 30 * 60
+
+# Setup Celery queues: default & long
+CELERY_DEFAULT_QUEUE = "default"
+CELERY_DEFAULT_EXCHANGE = "default"
+CELERY_DEFAULT_EXCHANGE_TYPE = "direct"
+CELERY_DEFAULT_ROUTING_KEY = "default"
+
+CELERY_QUEUES = (
+    Queue("default", Exchange("default"), routing_key="default"),
+    Queue("long-polling", Exchange("long-polling"), routing_key="long-polling"),
+)
+
+# Setup Celery routes for long-polling
+CELERY_TASK_ROUTES = {
+    "bptl.camunda.tasks.task_fetch_and_lock": {
+        "queue": "long-polling",
+        "exchange": "long-polling",
+        "routing_key": "long-polling",
+    }
+}
```

Comment:
The docs seem to advise against specifying this explicitly and instead relying on the automatic routing: https://docs.celeryproject.org/en/4.4.0/userguide/routing.html#routing-automatic

So this would then become something like:

```suggestion
# Setup Celery routes for long-polling
CELERY_TASK_ROUTES = {
    "bptl.camunda.tasks.task_fetch_and_lock": {
        "queue": "long-polling",
    },
    "bptl.camunda.tasks.task_schedule_new_fetch_and_lock": {
        "queue": "long-polling",
    },
}
```

I think this nicely hides away the Kombu/Exchange configuration where the default is fine for us.

Reply:
Prima - dan zetten we dus automatic routing aan.